src/components/try-hcp-callout/components/try-hcp-callout-prebuilt/content.ts

```diff
@@ -0,0 +1,55 @@
+import { TryHcpCalloutProps } from 'components/try-hcp-callout/types'
+import { ProductSlugWithContent } from '../../types'
+
+type HcpCalloutContent = Pick<
+	TryHcpCalloutProps,
+	'heading' | 'description' | 'ctaText' | 'ctaUrl'
+>
+
+export const tryHcpCalloutContent: Record<
+	ProductSlugWithContent,
+	HcpCalloutContent
+> = {
+	terraform: {
+		heading: 'Terraform Cloud',
+		description: 'Automate your infrastructure provisioning at any scale',
+		ctaText: 'Try Terraform Cloud for free',
+		ctaUrl: 'https://app.terraform.io/public/signup/account',
+	},
+	boundary: {
+		heading: 'HCP Boundary',
+		description: 'Securely connect to clouds and remote hosts',
+		ctaText: 'Try HCP Boundary',
+		ctaUrl: 'https://portal.cloud.hashicorp.com/sign-up',
+	},
+	consul: {
+		heading: 'HCP Consul',
+		description: 'Discover and securely connect your applications',
+		ctaText: 'Try HCP for free',
+		ctaUrl: 'https://portal.cloud.hashicorp.com/sign-up',
+	},
+	packer: {
+		heading: 'HCP Packer',
+		description: 'Automate build management across your cloud providers',
+		ctaText: 'Try HCP Packer',
+		ctaUrl: 'https://portal.cloud.hashicorp.com/sign-up',
+	},
+	vault: {
+		heading: 'HCP Vault',
+		description: 'Secure your applications and protect sensitive data',
+		ctaText: 'Try HCP Vault for free',
+		ctaUrl: 'https://portal.cloud.hashicorp.com/sign-up',
+	},
+	waypoint: {
+		heading: 'HCP Waypoint',
+		description: 'Simplify your application deployments across platforms',
+		ctaText: 'Try HCP Waypoint for free',
+		ctaUrl: 'https://portal.cloud.hashicorp.com/sign-up',
+	},
+	hcp: {
+		heading: 'HashiCorp Cloud Platform',
+		description: 'The fastest way to get up and running with HashiCorp tools',
+		ctaText: 'Try cloud for free',
+		ctaUrl: 'https://portal.cloud.hashicorp.com/sign-up',
+	},
```

Comment:
Can we double-check with design and make sure that these slightly different `ctaText`s are really what we want? It's interesting to me that some include "for free" and others don't, and some some say "Try HCP <Product Name>" but some others don't (aside from HCP and Terraform's -- those differing makes sense to me!) The compact version of some of these don't always match their non-compact version either, so just want to double check! ❤️

Reply:
Yep, I can double-check! I am going off what's in Figma, which is _in theory_ an approved source of truth, but 🤷 . I'll run the other updates in this PR, make sure Swingset is rendering the latest, & post in `#proj-dev-portal` 👍

Reply:
Fine with deferring this to a subsequent PR since it's not a prod change, just wanted to make sure to note the inconsistencies I saw! 😊

And AFAIK we're not adding these to the content of any pages, but rather just making it available as an MDX component & marketing content block to enable authors to place it where they want to when they're ready.

Reply:
☝️ Great Q re: MDX - I'm double-checking with Kelly on [this task](). In our handoff sync it sounded like the inverse: 
- We're adding the "default size" CTA to product landing pages (via `.json` authoring, but we're adding them in in specific spots and want to keep them there, so not exactly fully authorable)
- We're adding the "compact" CTA to sidecar on downloads views (JSX)
- We're _not_, at least from the latest info I have, intending to make `TryHcpCallout` available in MDX (?)

Reply:
> ☝️ Great Q re: MDX - I'm double-checking with Kelly on [this task](https://app.asana.com/0/0/1203365375583141/f). In our handoff sync it sounded like kind of the inverse:
> 
> * We're adding the "default size" CTA to product landing pages (via `.json` authoring, but we're adding them in in specific spots and want to keep them there, so not exactly fully authorable)
> * We're adding the "compact" CTA to sidecar on downloads views (JSX)
> * We're _not_, at least from the latest info I have, intending to make `TryHcpCallout` available in MDX (?)

To clarify on what I meant by "to enable authors to place it where they want to when they're ready" -- I was referring to what landing pages (not the location in each template) and us just adding the slot to the templates, not updating the JSON for authors.

It'd be good to get clarification on whether or not we need an MDX component, because [there's a comment from Kyle in the feedback doc](https://docs.google.com/document/d/1hAD6CV0VKxmJZLBdq1FpS_V1wCWXHCS2Eo5tkzzJ4cs/edit?disco=AAAAiVsmLfQ) that says we do want one (which is why I decided to create [a separate ticket for it](https://app.asana.com/0/0/1203365375583141/f)):

> i think we need two "authoring points" 1 is the primitive (like HCPCallout where we just pass 1 product slug prop). I think additionally is that we need to have support for it within our landing page .json as well. i connected with Kelly about this specifically and she DOES want to have the ability for authors to use them in both places